presto-plugin-toolkit/src/test/java/io/prestosql/plugin/base/security/TestFileBasedSystemAccessControl.java

```diff
@@ -268,4 +354,18 @@ private String getResourcePath(String resourceName)
     {
         return this.getClass().getClassLoader().getResource(resourceName).getPath();
     }
+
+    private static void assertDenied(ThrowingCallable callable, String expectedErrMessage)
```

Comment:
assertDenied  -> assertAccessDenied
expectedErrMessage -> expectedMessage

Reply:
This is changed.